packages/framework/tests/Feature/IncludesFacadeTest.php

```diff
@@ -113,4 +116,139 @@ public function testBladeReturnsRenderedDefaultValueWhenNotFound()
         $this->assertNull(Includes::blade('foo.blade.php'));
         $this->assertSame('default', Includes::blade('foo.blade.php', '{{ "default" }}'));
     }
+
+    public function testTorchlightAttributionIsNotInjectedToMarkdownPartials()
+    {
+        $this->file('resources/includes/without-torchlight.md', 'Syntax highlighted by torchlight.dev');
+
+        $this->assertSame(
+            '<p>Syntax highlighted by torchlight.dev</p>',
+            Includes::markdown('without-torchlight.md')
+        );
+    }
+
```

Comment:
This is obviously not working, did something wrong here. Also: Document in release notes

Reply:
Nvm, that's just to trigger the thing. Gonna clarify it.